### PR TITLE
Bump to Chisel 3.4 | Fix small deprecated items

### PR DESCRIPTION
**Pull request by abejgonzalez**
**Related issue**: https://github.com/ucb-bar/chipyard/pull/719

**Type of change**: other enhancement

**Impact**: no rtl change

**Development Phase**: proposal

**Release Notes**
Small changes to build with Chisel 3.4. We might want to specify how many TLB sets in addition to ways by default.

### CODE REVIEW

**Comment by abejgonzalez** [comment]
This will break CI until CY is also updated.

**Comment by jerryz123** [comment]
What is default TLB nSets set to? Is it 1?

**Comment by abejgonzalez** [comment]
> What is default TLB nSets set to? Is it 1?

Yup. Its set to 1 by default.

**Review by jerryz123 — APPROVED** [review]

File: .circleci/do-rtl-build.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ run "export RISCV=\"$TOOLS_DIR\"; \
      export VERILATOR_ROOT=\"$REMOTE_VERILATOR_DIR\"; \
      export COURSIER_CACHE=\"$REMOTE_WORK_DIR/.coursier-cache\"; \
      make -C $REMOTE_SIM_DIR clean; \
-     make -j$REMOTE_MAKE_NPROC -C $REMOTE_SIM_DIR JAVA_ARGS=\"$REMOTE_JAVA_ARGS\" ${mapping[$1]}"
+     make -j$REMOTE_MAKE_NPROC -C $REMOTE_SIM_DIR JAVA_OPTS=\"$REMOTE_JAVA_OPTS\" SBT_OPTS=\"$REMOTE_SBT_OPTS\"${mapping[$1]}"
```
Comment (abejgonzalez) [inline .circleci/do-rtl-build.sh]:
```suggestion
     make -j$REMOTE_MAKE_NPROC -C $REMOTE_SIM_DIR JAVA_OPTS=\"$REMOTE_JAVA_OPTS\" SBT_OPTS=\"$REMOTE_SBT_OPTS\" ${mapping[$1]}"
```

**Pull request merged**